src/index.css

```diff
@@ -163,7 +163,18 @@ input::placeholder {
   width: 100px;
   margin-bottom: 1rem;
 }
-
+.information.loading {
+  display: none;
+  max-height: 20px;
+  position: relative;
```

Comment:
Noyt sure if these are needed. Think you could just get away with `display: none`